semantic-python/test/fixtures/2-02-return-doesnt-translate.py

```diff
@@ -1,4 +1,4 @@
-# CHECK-JQ: .tree.contents[0][1].contents[1].contents.value.value == []
+# CHECK-TREE: #record { foo : foo = (\a -> a) }
```

Comment:
N.B. this is not quite right (it should be `{ foo = (\a -> a); #record { foo = foo }}`), but these tests have exposed some brokenness that the JSON tests weren’t catching. I’m going to fix this in another PR, because it entails some wider changes.